rllib/utils/exploration/epsilon_greedy.py

```diff
@@ -139,8 +140,9 @@ def _get_torch_exploration_action(self, q_values, explore, timestep):
             # Mask out actions, whose Q-values are -inf, so that we don't
             # even consider them for exploration.
```

Comment:
```suggestion
            # Mask out actions, whose Q-values are < FLOAT_MIN, so that we
            # don't even consider them for random exploration.
```